src/test/java/org/jabref/logic/cleanup/MoveFilesCleanupTest.java

```diff
@@ -117,15 +125,17 @@ public void movesFileFromSubfolderWithFileDirPattern() throws IOException {
         assertTrue(Files.exists(after));
 
         assertEquals(Optional
-                .of(FileFieldWriter.getStringRepresentation(new LinkedFile("", relativefileDir.toString(), ""))),
+                        .of(FileFieldWriter.getStringRepresentation(new LinkedFile("", relativefileDir.toString(), ""))),
                 entry.getField("file"));
     }
 
     @Test
-    public void movesFileFromSubfolderWithSubdirPattern() throws IOException {
+    public void movesFileFromSubfolderWithSubdirPattern(@TempDirectory.TempDir Path bibFolder) throws IOException {
         BibEntry local_entry = (BibEntry) entry.clone();
         local_entry.setField("year", "1989");
-        File subfolder = bibFolder.newFolder();
+        Path path = bibFolder.resolve("AnotherRandomlyNamedFolder");
+        Files.createDirectory(path);
+        File subfolder = path.toFile();
         File fileBefore = new File(subfolder, "test.pdf");
```

Comment:
Just one more minor thing I have before we can merge.
Please use the nio methods here and in the other things as well, e.g. instead of the old File = new File use  
Files.createFile(Path path)

Reply:
Are there other tests that have this issue? Or just change this test?